Website/src/rendering/src/stories/Checkout/OrderReviewDetails.stories.tsx

```diff
@@ -4,80 +4,34 @@ import { ComponentStory, ComponentMeta } from '@storybook/react';
 import OrderReviewDetails from '../../components/Checkout/OrderReviewDetails';
 import { MockStore } from '../mock-store';
 import { cartState, productCacheSlice, loggedInAuthSlice, shipMethods } from './CheckoutCommon';
-import { getMockExpirationDate } from '../utils';
+import { LineItem } from 'ordercloud-javascript-sdk';
 
 export default {
   title: 'Components/Checkout/OrderReviewDetails',
   component: OrderReviewDetails,
 } as ComponentMeta<typeof OrderReviewDetails>;
 
-const Template: ComponentStory<typeof OrderReviewDetails> = (args) => (
-  <OrderReviewDetails {...args} />
-);
-
-const shippingAddress = {
-  ID: 'mockaddressid',
-  AddressName: 'Marty Byrde Home',
-  Street1: '6818 Gaines Ferry Road',
-  City: 'Flowery Branch',
-  State: 'GA',
-  Zip: '30542',
-  Country: 'US',
-};
-
-const shipEstimateResponse = {
-  ShipEstimates: [
-    {
-      ID: 'STATIC_SINGLE_SHIPMENT',
-      SelectedShipMethodID: 'EXPRESS_DELIVERY',
-      ShipMethods: shipMethods,
-    },
-  ],
-};
-
-const billingAddress = {
-  ID: 'mockaddressid',
-  AddressName: 'Marty Byrde Home',
-  Street1: '6818 Gaines Ferry Road',
-  City: 'Flowery Branch',
-  State: 'GA',
-  Zip: '30542',
-  Country: 'US',
-};
-
-const payments = [
-  {
-    ID: 'mockpaymentid',
-    Type: 'CreditCard',
-    CreditCardID: 'mock-creditcard-id',
-    Accepted: true,
-    Amount: 100,
-    xp: {
-      CreditCard: {
-        ID: 'mockcreditcardid',
-        CardType: 'Visa',
-        CardholderName: 'Jon Snow',
-        PartialAccountNumber: '6123',
-        ExpirationDate: getMockExpirationDate(),
-      },
-    },
-  },
-];
+const Template: ComponentStory<typeof OrderReviewDetails> = () => <OrderReviewDetails />;
 
 export const Default = Template.bind({});
 const defaultCartState = {
   ...cartState,
-  shippingAddress,
-  shipEstimateResponse,
-  payments,
+  shipEstimateResponse: {
+    ShipEstimates: [
+      {
+        ID: 'STATIC_SINGLE_SHIPMENT',
+        SelectedShipMethodID: 'EXPRESS_DELIVERY',
+        ShipMethods: shipMethods,
+      },
+    ],
+  },
   order: {
-    ID: 'mockorderid',
+    ...cartState.order,
     Subtotal: 1069.98,
     ShippingCost: 4.99,
     TaxCost: 256.79,
     Total: 1331.76,
     LineItemCount: 4,
```

Comment:
Can we move this:
```
Subtotal: 1069.98,
ShippingCost: 4.99,
TaxCost: 256.79,
Total: 1331.76,
LineItemCount: 4,
xp: {
  DeliveryType: 'Ship',
},
```
to CheckoutCommon and override the different in the NoLineItems variant since the other two are identical?